examples/js/loaders/GLTFLoader.js

```diff
@@ -25,10 +25,13 @@ THREE.GLTFLoader = ( function () {
 
 			var scope = this;
 
-			var path = this.path !== undefined ? this.path : THREE.LoaderUtils.extractUrlBase( url );
+			var path = this.resourcePath !== undefined
+				? this.resourcePath
+				: THREE.LoaderUtils.extractUrlBase( url );
```

Comment:
Looking at other PRs (like https://github.com/mrdoob/three.js/pull/14941, https://github.com/mrdoob/three.js/pull/14932) I think the logic should be equivalent to:

```js
var resourcePath = this.resourcePath || this.path || THREE.LoaderUtils.extractUrlBase( url );
```

The code currently in this PR would ignore `this.path` and fall back to extracting a base path from the URL.

Reply:
That looks reasonable. Updated.